modules/osdk-ansible-managing-cr-status.adoc

```diff
@@ -0,0 +1,106 @@
+----
+status:
+  conditions:
+    - ansibleResult:
+      changed: 3
+      completion: 2018-12-03T13:45:57.13329
+      failures: 1
+      ok: 6
+      skipped: 0
+    lastTransitionTime: 2018-12-03T13:45:57Z
+    message: 'Status code was -1 and not [200]: Request failed: <urlopen error [Errno
+      113] No route to host>'
+    reason: Failed
+    status: "True"
+    type: Failure
+  - lastTransitionTime: 2018-12-03T13:46:13Z
+    message: Running reconciliation
+    reason: Running
+    status: "True"
+    type: Running
+----
+
+Ansible-based Operators also allow Operator authors to supply custom status
+values with the
+link:https://github.com/fabianvf/ansible-k8s-status-module[`k8s_status`] Ansible
+module. This allows the author to update the `status` from within Ansible with
+any key-value pair as desired.
+
+By default, Ansible-based Operators always include the generic Ansible
+run output as shown above. If you would prefer your application did _not_ update
+the status with Ansible output, you can track the status manually
+from your application.
+
+.Procedure
+
+. To track CR status manually from your application, update the Watches file
+with a `manageStatus` field set to `false`:
++
+[source,yaml]
+----
+- version: v1
+  group: api.example.com
+  kind: Foo
+  role: /opt/ansible/roles/Foo
+  manageStatus: false
+----
+
+. Then, use the `k8s_status` Ansible module to update the subresource. For
+example, to update with key `foo` and value `bar`, `k8s_status` can be used as
+shown:
++
+[source,yaml]
+----
+- k8s_status:
+  api_version: app.example.com/v1
+  kind: Foo
+  name: "{{ meta.name }}"
+  namespace: "{{ meta.namespace }}"
+  status:
+    foo: bar
+----
+
+.Additional resources
+
+- For more details about user-driven status management from Ansible-based
+Operators, see the
+link:https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/ansible-operator-status.md[Ansible Operator Status Proposal].
+
+[id='osdk-ansible-managing-cr-status-locally-{context}']
+== Using the k8s_status Ansible module when testing locally
+
+If your Operator takes advantage of the `k8s_status` Ansible module and you want
+to test the Operator locally with the `operator-sdk up local` command, you must
+install the module in a location that Ansible expects. This is done with the
+`library` configuration option for Ansible.
+
+For this example, assume the user is placing third-party Ansible modules in the
+`/usr/share/ansible/library/` directory.
+
+.Procedure
+
+. To install the `k8s_status` module, set the `ansible.cfg` file to search in
+the `/usr/share/ansible/library/` directory for installed Ansible modules:
++
+----
+$ echo "library=/usr/share/ansible/library/" >> /etc/ansible/ansible.cfg
+----
+
+. Add the `k8s_status.py` file to the `/usr/share/ansible/library/` directory:
++
+----
+$ wget https://raw.githubusercontent.com/fabianvf/ansible-k8s-status-module/master/k8s_status.py -O /usr/share/ansible/library/k8s_status.py
```

Comment:
I think we should use this link instead: https://raw.githubusercontent.com/openshift/ocp-release-operator-sdk/master/library/k8s_status.py and update that to the release branch when it is created.